linux_sql/README.md

```diff
@@ -0,0 +1,48 @@
+
+# Introduction
+The Objective of this project is to design an MVP in order to gather hardware specifications and resource usage of each node in a Linux cluster. In Jarvis, the Linux cluster administration team manages a Linux cluster of 10 nodes running Centos7. These nodes are internally connected through a switch and able to communicate through internal IPv4 addresses. The administration team has requested a bash agent program that is installed on each node and gathers the requested data. The collected data is inserted into a Postgres database instance which is pulled from Docker Hub. Moreover, the LCA team will use the data to generate reports for future resource planning purposes. In order to make decision making easier, a queries file has been created for the administration team to see what nodes have the highest memory and number of CPUs, what is their usage over a 5 minute interval and detecting failures. 
+
+# Quick Start
```

Comment:
Please add a bit more detail here: .e.g. the crontab instruction is not clear. This should read like instructions: it does not have to be long or particularly verbose, but it should be clear why and how each command is run.